preset/index.js

```diff
@@ -19,7 +19,7 @@ const preset = {
     },
   },
   icons: {
-    iconfont: 'fa',
```

Comment:
See https://vuetifyjs.com/en/customization/icons/#install-font-awesome-4-icons for details